azure-mediaservices-spring-boot-starter/README.md

```diff
@@ -22,6 +22,15 @@ azure.mediaservices.account-name=put-your-media-services-account-name-here
 azure.mediaservices.account-key=put-your-media-services-account-key-here
 ```
 
+# Optional
+If you are using network proxy then add below properties to run azure media services behind proxy
```

Comment:
Yeah I am user of Media Services, my client has a strict Firewall policy . we cannot communicate to Azure Media Services directly and it can be done only using network proxy provided by client